resources/eventing/values.yaml

```diff
@@ -36,7 +36,7 @@ postUpgradeJob:
     name: event-sources-upgrade-job
     tag: PR-10733
```

Comment:
Please update the `postUpgradeJob.image.tag` with the final image (if available).

Reply:
Fixed